src/app/pages/record/add-event/add-event.component.ts

```diff
@@ -12,14 +14,12 @@ import { RecordService } from '../record.service';
 })
 export class AddEventComponent implements OnInit, OnDestroy {
 
-
   public form!: FormGroup;
-
   private sub: Subscription = new Subscription();
 
   constructor(
     @Inject(MAT_DIALOG_DATA) public data: ICategories[],
-    public dialogRef: MatDialogRef<any>,
+    public dialogRef: MatDialogRef<RecordComponent, HistoryComponent>,
     private recordService: RecordService) { }
 
   public ngOnInit(): void {
```

Comment:
remove unused constructor and ngOnInit

Reply:
also imports, variables and other things

Reply:
I use constructor and ngOnInit everywhere, and variavles too